src/components/charts/CandleChartContainer.vue

```diff
@@ -171,115 +121,91 @@ onMounted(() => {
-            <span class="text-nowrap">Show Chart Areas</span>
-          </BaseCheckbox>
-          <BaseCheckbox v-model="settingsStore.useHeikinAshiCandles">
-            <span class="text-nowrap">Heikin Ashi</span>
-          </BaseCheckbox>
-
-          <PlotConfigSelect></PlotConfigSelect>
-
-          <div class="me-0 md:me-1">
-            <Button
-              size="small"
-              title="Plot configurator"
-              severity="secondary"
-              @click="showConfigurator"
-            >
-              <template #icon>
-                <i-mdi-cog width="12" height="12" />
-              </template>
-            </Button>
-          </div>
-        </div>
       </div>
-      <div class="h-full flex">
+      <div v-if="botStore.activeBot.plotMultiPairs.length == 1" class="h-full flex">
         <div class="min-w-0 w-full flex-1">
-          <CandleChart
-            v-if="hasDataset"
-            :dataset="dataset"
-            :trades="trades"
-            :plot-config="plotStore.plotConfig"
-            :heikin-ashi="settingsStore.useHeikinAshiCandles"
-            :show-mark-area="settingsStore.showMarkArea"
-            :use-u-t-c="settingsStore.timezone === 'UTC'"
-            :theme="settingsStore.chartTheme"
-            :slider-position="sliderPosition"
-            :color-up="colorStore.colorUp"
-            :color-down="colorStore.colorDown"
-            :start-candle-count="settingsStore.chartDefaultCandleCount"
-            :label-side="settingsStore.chartLabelSide"
-          />
-          <div v-else class="m-auto">
-            <ProgressSpinner v-if="isLoadingDataset" class="w-5 h-5" label="Spinning" />
-            <div v-else class="text-lg">
-              {{ noDatasetText }}
-            </div>
-            <p v-if="botStore.activeBot.historyTakesLonger">
-              This is taking longer than expected ... Hold on ...
-            </p>
-          </div>
+          <SingleCandleChartContainer
+            :available-pairs="availablePairs"
+            :pair="botStore.activeBot.plotMultiPairs[0]"
+            :historic-view="botStore.activeBot.isWebserverMode"
+            :timeframe="timeframe"
+            :trades="props.trades"
+            :strategy="props.strategy"
+            :slider-position="props.sliderPosition"
+            :is-single-pair-view="true"
+            @refresh-data="refresh()"
+          >
+          </SingleCandleChartContainer>
         </div>
-        <Transition name="fade">
-          <div
-            v-if="!plotConfigModal"
-            v-show="showPlotConfig"
-            class="grow border rounded-md ps-1 border-surface-300 dark:border-surface-700 max-w-110"
+      </div>
+      <div v-else class="flex flex-wrap gap-4 mt-2">
+        <div
+          v-for="pair in botStore.activeBot.plotMultiPairs"
+          :key="pair"
+          class="flex-[1_1_30%] p-4"
+        >
+          <div class="font-bold mt-2 mb-1">{{ pair }}</div>
```

Comment:
If you move this header into the singleChartContainer - you can save (vertical) space by rendering it at the same height than the "long/short entries" line - which currently wastes Tons of space vertically.